src/main/java/Friday.java

```diff
@@ -0,0 +1,41 @@
+import commands.Command;
```

Comment:
Perhaps putting the class into a package (as per the course's coding standards)? I would suggest creating a `friday` package, that would contain the `Friday` class as well as the other packages!

Reply:
Agreed! Thank you!